--- multi-gnome-terminal-1.4.0/gnome-terminal/enhanced_gui.c~	Fri Apr 26 19:48:12 2002
+++ multi-gnome-terminal-1.4.0/gnome-terminal/enhanced_gui.c	Mon Apr 29 20:14:27 2002
@@ -3517,7 +3517,7 @@
     	      if (cfg->termname && cfg->termname [0])
 		env_copy [term_pos] = cfg->termname;
 	      else
-		env_copy [term_pos] = "TERM=xterm";
+		env_copy [term_pos] = "TERM=xterm-color";
 
 	      if (cmd){
 		environ = env_copy;
@@ -5652,7 +5652,7 @@
 	    if (cfg->termname && cfg->termname [0])
 	      env_copy [term_pos] = cfg->termname;
 	    else
-	      env_copy [term_pos] = "TERM=xterm";
+	      env_copy [term_pos] = "TERM=xterm-color";
 
 	    if (cmd){
 	      environ = env_copy;
--- multi-gnome-terminal-1.4.0/gnome-terminal/multi-gnome-terminal.c~	Fri Apr 26 19:46:25 2002
+++ multi-gnome-terminal-1.4.0/gnome-terminal/multi-gnome-terminal.c	Mon Apr 29 20:15:03 2002
@@ -5100,11 +5100,11 @@
 		cfg->invoke_as_login_shell = gnome_config_get_bool("invoke_as_login_shell");
 		{
 			char *ctmp;
-			ctmp = gnome_config_get_string("termname=xterm");
+			ctmp = gnome_config_get_string("termname=xterm-color");
 			if(ctmp && *ctmp)
 				cfg->termname = g_strconcat("TERM=", ctmp, NULL);
 			else
-				cfg->termname = g_strdup("TERM=xterm");
+				cfg->termname = g_strdup("TERM=xterm-color");
 			g_free(ctmp);
 		}
 		cfg->window_title = gnome_config_get_string("window_title");
@@ -5240,7 +5240,7 @@
 			if(ctmp) ctmp++;
 		}
 		if(!ctmp)
-			ctmp = "xterm";
+			ctmp = "xterm-color";
 		gnome_config_set_string("termname", ctmp);
 
 		gnome_config_set_string ("window_title",
