--- multi-gnome-terminal-1.5.2/gnome-terminal/zvtterm.c.orig	Sat Jul 20 13:34:57 2002
+++ multi-gnome-terminal-1.5.2/gnome-terminal/zvtterm.c	Tue Sep  3 14:10:07 2002
@@ -3775,11 +3775,13 @@
     break;
 
   case GDK_F1:  case GDK_F2:  case GDK_F3:  case GDK_F4:
+#ifdef XTERM_WITH_VTXXX_FKEYS
     if (event->state & GDK_SHIFT_MASK){
        p+=sprintf (p, "\033[%d~", f1_f20_remap[10+event->keyval-GDK_F1]);
     } else
        p+=sprintf (p, "\033O%c", 'P' + (event->keyval-GDK_F1));
      break;
+#endif
 #else
   case GDK_KP_F1:
   case GDK_F1:
