diff -Nur multi-gnome-terminal-1.5.0.orig/gnome-terminal/enhanced_gui.c multi-gnome-terminal-1.5.0/gnome-terminal/enhanced_gui.c
--- multi-gnome-terminal-1.5.0.orig/gnome-terminal/enhanced_gui.c	Thu Jun 13 18:02:19 2002
+++ multi-gnome-terminal-1.5.0/gnome-terminal/enhanced_gui.c	Fri Jun 28 00:49:58 2002
@@ -4077,7 +4077,7 @@
     	      if (cfg->termname && cfg->termname [0])
 		env_copy [term_pos] = cfg->termname;
 	      else
-		env_copy [term_pos] = "TERM=xterm";
+		env_copy [term_pos] = "TERM=xterm-color";
 
 	      if (cmd){
 		environ = env_copy;
@@ -6548,7 +6548,7 @@
 	    if (cfg->termname && cfg->termname [0])
 	      env_copy [term_pos] = cfg->termname;
 	    else
-	      env_copy [term_pos] = "TERM=xterm";
+	      env_copy [term_pos] = "TERM=xterm-color";
 
 	    if (cmd){
 	      environ = env_copy;
@@ -7140,7 +7140,7 @@
 	      if (cfg->termname && cfg->termname [0])
 		env_copy [term_pos] = cfg->termname;
 	      else
-		env_copy [term_pos] = "TERM=xterm";
+		env_copy [term_pos] = "TERM=xterm-color";
 
 	      if (cmd){
 		environ = env_copy;
diff -Nur multi-gnome-terminal-1.5.0.orig/gnome-terminal/multi-gnome-terminal.c multi-gnome-terminal-1.5.0/gnome-terminal/multi-gnome-terminal.c
--- multi-gnome-terminal-1.5.0.orig/gnome-terminal/multi-gnome-terminal.c	Thu Jun 13 21:34:28 2002
+++ multi-gnome-terminal-1.5.0/gnome-terminal/multi-gnome-terminal.c	Fri Jun 28 00:48:36 2002
@@ -5628,11 +5628,11 @@
     cfg->invoke_as_login_shell = gnome_config_get_bool("invoke_as_login_shell");
       {
 	char *ctmp;
-	ctmp = gnome_config_get_string("termname=xterm");
+	ctmp = gnome_config_get_string("termname=xterm-color");
 	if(ctmp && *ctmp)
 	  cfg->termname = g_strconcat("TERM=", ctmp, NULL);
 	else
-	  cfg->termname = g_strdup("TERM=xterm");
+	  cfg->termname = g_strdup("TERM=xterm-color");
 	g_free(ctmp);
       }
     cfg->window_title = gnome_config_get_string("window_title");
@@ -5768,7 +5768,7 @@
       if(ctmp) ctmp++;
     }
     if(!ctmp)
-      ctmp = "xterm";
+      ctmp = "xterm-color";
     gnome_config_set_string("termname", ctmp);
 
     gnome_config_set_string ("window_title",
